app/src/main/kotlin/com/simplemobiletools/flashlight/helpers/MyCameraImpl.kt

```diff
@@ -7,8 +7,13 @@ import com.simplemobiletools.commons.extensions.toast
 import com.simplemobiletools.flashlight.R
 import com.simplemobiletools.flashlight.extensions.config
 import com.simplemobiletools.flashlight.extensions.updateWidgets
-import com.simplemobiletools.flashlight.models.Events
-import org.greenrobot.eventbus.EventBus
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.flow.MutableSharedFlow
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asSharedFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.launch
 
 class MyCameraImpl private constructor(val context: Context, private var cameraTorchListener: CameraTorchListener? = null) {
```

Comment:
Why are the constructor params not private?

Reply:
Not sure, probably somebody forgot to make them private.

Reply:
Well can we make em private?

Reply:
Yeah, I did that already.